<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Progress Quest 7</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../grumdrig-pq-55d918524976/crossed_swords_sm.gif</normaloff>../../../grumdrig-pq-55d918524976/crossed_swords_sm.gif</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="3">
     <widget class="QFrame" name="frm_story">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="lbl_plot">
         <property name="text">
          <string>Plot Development</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lst_plot">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="viewMode">
          <enum>QListView::ListMode</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="pb_plot">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_quests">
         <property name="text">
          <string>Quests</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lst_quests"/>
       </item>
       <item>
        <widget class="QProgressBar" name="pb_quest">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QFrame" name="frm_inventory">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="lbl_equipment">
         <property name="text">
          <string>Equipment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tbl_equipment">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::NoPen</enum>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>80</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>12</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>6</number>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Peice</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Equiped</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_inventory">
         <property name="text">
          <string>Inventory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tbl_inventory">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::NoPen</enum>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>180</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>12</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>6</number>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Item</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_encumbrance">
         <property name="text">
          <string>Encumbrance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="pb_encumbrance">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QFrame" name="frm_action">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbl_action">
         <property name="text">
          <string>Action</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="pb_action">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frm_char">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lbl_char">
         <property name="text">
          <string>Character Sheet</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tbl_traits">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>96</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::NoPen</enum>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>80</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>12</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>6</number>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Traits</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tbl_stats">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::NoPen</enum>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>140</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>12</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>6</number>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Stat</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="background">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_xp">
         <property name="text">
          <string>Experience</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="pb_experience">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_spells">
         <property name="text">
          <string>Spell Book</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tbl_spells">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::NoPen</enum>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>140</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>12</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>6</number>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>spell</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>level</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
